Library/TimerLabelTests.swift

```diff
@@ -0,0 +1,83 @@
+import XCTest
+import Library
+
+final class TimerLabelTests: XCTestCase {
+
+    private weak var timer: Timer!
+
+    override func setUp() {
+        let mockScheduled = { [weak self] (interval: TimeInterval, repeats: Bool, block: @escaping (Timer) -> Void) -> Timer in
+            /*
+             This time interval is okay since we will be manually firing the timer,
+             and don't want it to fire on its own. We need to create a Timer
+             this way because `Timer()` will not return a timer unfortunately.
+             (something to explore more, but this solution is good enough for a test)
+             */
+            let t = Timer(timeInterval: 100, repeats: true, block: block)
+            self?.timer = t
+            return t
+        }
+        Current = AppEnvironment(scheduledTimer: mockScheduled)
+    }
+
+    override func tearDown() {
+        timer?.invalidate()
+    }
+
+    func testTimerStartAtZero() {
+        let timerLabel = TimerLabel(frame: .zero)
+        timerLabel.start(at: 0)
+        guard let timerText = timerLabel.text else {
+            XCTFail()
+            return
+        }
+        XCTAssertEqual(timerText, "00:00:00")
+    }
+
+    func testChangeDisplayTime() {
+        let timerLabel = TimerLabel(frame: .zero)
+        timerLabel.changeDisplayTime(time: 11111)
+        guard let timerText = timerLabel.text else {
+            XCTFail()
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/jasonzurita/BabyPatterns/blob/7d4b7a47f520e10c677d21891d119ba0b38f461e/Library/TimerLabelTests.swift#L41"></span>An XCTFail call should include a description of the assertion.<br />
<code>xctfail_message</code> <code>TimerLabelTests.swift:41</code></td>
    </tr>
  </tbody>
</table>